src/core/MOM_CoriolisAdv.F90

```diff
@@ -73,6 +76,10 @@ module MOM_CoriolisAdv
                              !! relative to the other one is used.  This is only
                              !! available at present if Coriolis scheme is
                              !! SADOURNY75_ENERGY.
+  logical, public :: USE_WENO !< If WENOVI7TH_PV_ENSTRO and WENOVI7TH_ENSTRO schemes are used,
```

Comment:
We generally discourage the introduction of public elements in an opaque type unless the alternatives would be overly cumbersome.  I do not think that there is a need for a public element here. There is an analogous issue with needing the stencil size for the continuity solver, which has been addressed by the introduction of the function `continiuty_stencil()` in MOM_continuity.F90.  Please follow that code as a pattern to obtain the CoriolisAdv stencil.